scripts/create-github-release.js

```diff
@@ -29,21 +29,31 @@ async function createGitHubRelease() {
   }
 
   const octokit = new Octokit({
-    auth: `token ${process.env.GITHUB_TOKEN}`,
+    auth: process.env.GITHUB_TOKEN,
   });
 
   const changes = await changelistSinceTag(octokit);
-
   const [owner, repo] = pkg.repository.split('/');
+  const releaseName = `v${pkg.version}`;
+
+  console.log(
+    `Creating release ${releaseName} of ${owner}/${repo} with changelog:
+
+${changes}
+`
+  );
 
   await octokit.repos.createRelease({
+    // Required params.
```

Comment:
No changes to the params here, I've just grouped them into required and optional ones.